zetaclient/bitcoin_client.go

```diff
@@ -293,35 +306,34 @@ func (ob *BitcoinChainClient) ConfirmationsThreshold(amount *big.Int) int64 {
 }
 
 // returns isIncluded, isConfirmed, Error
-func (ob *BitcoinChainClient) IsSendOutTxProcessed(sendHash string, nonce int, _ common.CoinType, logger zerolog.Logger) (bool, bool, error) {
-	outTxID := ob.GetTxID(uint64(nonce))
+func (ob *BitcoinChainClient) IsSendOutTxProcessed(sendHash string, nonce uint64, _ common.CoinType, logger zerolog.Logger) (bool, bool, error) {
+	outTxID := ob.GetTxID(nonce)
```

Comment:
Not required of this PR , but good to have 
It would be useful to have more documentation for TXID , in the GetTxID function

Reply:
Great great. I put it in backlog.